modules/message/html/tag.js

```diff
@@ -0,0 +1,17 @@
+var { h } = require('mutant')
```

Comment:
This file is in the wrong folder.
Should be in modules/message/hmtl/action/tag.js